dapps/profile.vue/src/components/sharings/sharings.vue

```diff
@@ -34,11 +34,9 @@ the following URL: https://evan.network/license/
           class="light"
           alignment="right"
           ref="shareSidebar"
-          panelId="shareSidebar"
           :showBackdrop="windowWidth < 1200"
           :mountId="windowWidth < 1200 ? null : 'dapp-wrapper-sidebar-right'"
-          :isOpen="windowWidth >= 1200 || selectedSharedContacts.length > 0"
```

Comment:
I think we need to keep this. Could also result in conflicts, since Daniel is also working currently on this logic...